app/assets/stylesheets/elements/_standout.scss

```diff
@@ -89,15 +89,15 @@
     }
   }
 
-  small {
+  &__subtitle {
```

Comment:
While this syntax works, it makes it very difficult to search-all for when debugging. Preferably it would be it's own full class.